spec.bs

```diff
@@ -2443,13 +2478,16 @@ of the following global objects:
 
 <div algorithm>
   To <dfn>evaluate a script</dfn> with a [=ECMAScript/realm=] |realm|, [=string=] |script|, [=string=]
-  |functionName|, a [=list=] |arguments|, and an integer millisecond duration |timeout|, run these steps.
+  |functionName|, a [=list=] |arguments|, and an integer millisecond [=duration=] |timeout|, run these steps.
   They return a [=ECMAScript/Completion Record=], which is either an [=ECMAScript/abrupt completion=] (in
   the case of a parse failure or execution error), or a [=ECMAScript/normal completion=] populated with the
   [=ECMAScript/ECMAScript language value=] result of invoking |functionName|.
 
     1. [=Assert=] that these steps are running [=in parallel=].
 
+    1. If |timeout| &le; 0, [=immediately=] interrupt the execution and set |finalCompletion| to a

```

Comment:
It seems like this should not do anything `[=immediately=]`, but should just set `finalCompletion` to a throw completion and jump to the return steps like the other return steps in this algorithm, right?